src/ddmd/dsymbolsem.d

```diff
@@ -4744,9 +4744,13 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
             if (funcdecl.isOverride)
             {
                 if (funcdecl.storage_class & STCdisable)
-                    funcdecl.deprecation("overridden functions cannot be annotated @disable");
+                    deprecation(funcdecl.loc,
+                                "`%s` cannot be annotated with @disable because it is overriding a function in the base class",
```

Comment:
In general, all code snippets in error messages must be surrounded by back-ticks, in order to get syntax highlighting.
Please also add back-ticks around `@disable` and `deprecated`.

Reply:
Indeed, thanks!